app/src/main/java/org/schabi/newpipe/player/playback/MediaSourceManager.java

```diff
@@ -146,6 +150,20 @@ private MediaSourceManager(@NonNull final PlaybackListener listener,
                     + " ms] for them to be useful.");
         }
 
+        if (DEBUG) {
+            final var currentItem = playQueue.getItem();
+
+            if (currentItem != null) {
+                Log.d(TAG, "Creating MediaSourceManager["
+                           + StreamingServiceId.nameFromId(currentItem.getServiceId())
```

Comment:
Use this instead (also in other places) https://github.com/TeamNewPipe/NewPipe/blob/2a9c6f05387866bef2c42ab67226c14936917d27/app/src/main/java/org/schabi/newpipe/util/ServiceHelper.java#L138